examples/tensorflow/classification/main.py

```diff
@@ -170,12 +176,12 @@ def run(config):
                 return_dict=True)
             uncompressed_model_accuracy = 100 * results['acc@1']
 
+    compression_state = None
+    if resume_training:
+        compression_state = extract_compression_state(config.ckpt_path)
```

Comment:
what do you think about making it more shorter by checking path for the None inside function?
resume_training won't be needed, because it's already had this check `resume_training = config.ckpt_path is not None`

Reply:
It is possible but we need a flag to check this
https://github.com/ljaljushkin/nncf_pytorch/blob/d516aab6466ef815d1c503dfeb0a860ac797cd6d/examples/tensorflow/classification/main.py#L217-L220

Reply:
but eventually it's checking the same thing
```
resume_training = config.ckpt_path is not None
```